routes/cities.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const connection = require('../config')
+const router = express.Router();
+
+// GET all cities
+router.get('/', (req, res) => {
+    connection.query('SELECT * FROM city', (err, result)=> {
+        console.log(err)
+        if (err) {
+            res.status(500).json('Erreur lors de la récupération des villes')
+        } else {
+            res.status(200).json(result)
+        }
+    })
+})
+
+// POST a city
+router.post('/', (req, res) => {
+    connection.query('INSERT INTO city SET ?', [req.body], (err, result) => {
```

Comment:
Pour plus de lisibilité, je pense qu'il faut ajouter une variable d'environnement. 
```suggestion
router.post('/', (req, res) => {
    const formBody = req.body;
    connection.query('INSERT INTO city SET ?', [formBody], (err, result) => {
```